packages/grid/x-data-grid-pro/src/tests/columnPinning.DataGridPro.test.tsx

```diff
@@ -116,6 +116,7 @@ describe('<DataGridPro /> - Column pinning', () => {
     fireEvent.mouseDown(separator, { clientX: 100 });
     fireEvent.mouseMove(separator, { clientX: 110, buttons: 1 });
     fireEvent.mouseUp(separator);
+    clock.tick(0);
```

Comment:
After fixing the issue, this test started to fail, even though the feature itself worked just fine.

Since the `useCallback` was added, render zone refresh wasn't triggered that often and for test to pass it was necessary to wait for the actual resize a bit.

Reply:
Yeah, there's a `setTimeout` inside the column resize hook that was running.

```suggestion
    clock.runToLast();
```